app/assets/images/payment_icons/MTN.svg

```diff
@@ -0,0 +1 @@
34m0 .35a.74.74 0 00.79-.73.71.71 0 00-.76-.69.75.75 0 00-.8.73.72.72 0 00.77.69m-1.45-1.84l-.46.7-.51-.7h-.45v1.78h.44v-1l.36.48h.26l.32-.48v1h.49v-1.78zm-2.92 0v1l-.8-1h-.45v1.78h.45v-1l.84 1h.4v-1.78zm-2.58.4v1.38h.5v-1.38h.58v-.4H14v.4zm-1.28-.4l-.46.7-.51-.7h-.45v1.78h.45v-1l.35.48H13l.32-.48v1h.49v-1.78z"/><path class="a" d="M32 13.4V4.66a.67.67 0 00-.7-.66H6v13.55h21.8a.64.64 0 00.48-.2l3.47-3.47a.67.67 0 00.2-.48"/><path class="d" d="M31.3 4.66H19.59v12.21h8.23l3.48-3.48V4.66z"/><path class="a" d="M28.36 4.65h-2.07a4.92 4.92 0 00-1.72-1.95.24.24 0 00-.14-.05.32.32 0 00-.26.1l-.7.79a6.09 6.09 0 00-1.23-.39s-.17 0-.24.08c-.21.33-1.91 3.47-2 3.59s-.25.47 0 .54a16.61 16.61 0 013.58 1.09v.39l-1.06 5.89a.5.5 0 00.36.56s-.14-.06 3.19 1a.34.34 0 00.46-.21l2.12-10.89c.1-.44-.12-.53-.29-.54"/><path class="d" d="M6.7 4.66h12.22v12.22H6.7z"/><path class="c" d="M12.81 13.14c3.18 0 5.77-1.06 5.77-2.37S16 8.39 12.81 8.39 7 9.45 7 10.77s2.58 2.37 5.77 2.37"/><path class="d" d="M14.25 10.18l.15-.6h-1.97l-.15.6h.65l-.32 1.26h.68l.32-1.26h.64z"/><path class="a" d="M12.26 9.58h-.98l-.63 1.38V9.58H9.7l-.59 2.37h.65l.39-1.53v1.53h.5l.75-1.53-.36 1.53h.63l.59-2.37zm4.24 0h-.62l-.33 1.26-.29-1.26h-.69l-.6 2.37h.63l.33-1.29.33 1.29h.65l.59-2.37z"/><path class="e" d="M12.57 11.61l-.09.34h.68l.08-.34h-.67z"/><path class="c" d="M24.37 2.94l-.64.72a5.38 5.38 0 012.15 2 5.45 5.45 0 01.37.68 6.63 6.63 0 01.46 1.2 8.8 8.8 0 00-.12-1.06 5.09 5.09 0 00-2.16-3.53h-.06"/><path d="M26.73 5.84a21.7 21.7 0 01.48 4.85l1-5.21C28.29 5.1 28.09 5 28 5h-1.56a5.79 5.79 0 01.29.86"/><path class="e" d="M23.86 10.72l2.91.73c-.23-1.28-1.58-2.23-3-2.89a7.72 7.72 0 010 2.16"/><path d="M29.4 5.08l-.48-.48a.79.79 0 00-.56-.23h-2.22l.15.29h2.07c.17 0 .39.1.29.54l-2.06 10.86a.34.34 0 01-.46.21c-3.33-1-3.19-1-3.19-1a.5.5 0 01-.36-.56l1.06-5.89v-.39l-.17-.07-1.14 6.3a.79.79 0 00.56.89l3.43 1a.85.85 0 00.22 0 .82.82 0 00.31-.07l.62-.26a.78.78 0 00.47-.6l1.71-10a.79.79 0 00-.22-.69"/><path d="M25.31 12.48a.23.23 0 00.17.25l1 .25a.21.21 0 00.26-.15l.08-.43a.23.23 0 00-.18-.25l-1-.24a.19.19 0 00-.24.15zM23.42 12a.22.22 0 00.16.24l.92.24a.19.19 0 00.24-.15l.08-.41a.23.23 0 00-.17-.25l-.92-.22a.18.18 0 00-.23.14zm1.64 1.65a.23.23 0 00.17.25l1 .27a.19.19 0 00.25-.14l.08-.43a.23.23 0 00-.18-.26l-1-.26a.19.19 0 00-.24.14zm-1.89-.56a.22.22 0 00.16.25l.92.25a.18.18 0 00.24-.14l.08-.41a.23.23 0 00-.17-.25l-.91-.25a.2.2 0 00-.24.14zm1.65 1.73a.24.24 0 00.17.26l1 .29a.19.19 0 00.25-.14l.08-.43a.25.25 0 00-.18-.27l-1-.27a.19.19 0 00-.24.14zm-1.88-.58a.23.23 0 00.16.25l.91.27a.19.19 0 00.24-.14l.08-.42a.22.22 0 00-.16-.25l-.92-.26a.18.18 0 00-.23.13z"/><path class="c" d="M22.17 3.43L20.15 7c.86.12 5.67 1.26 6.65 3.54a1.74 1.74 0 01.1.23v-.25a12.14 12.14 0 00-.9-4.11 5.75 5.75 0 00-.36-.65 5.1 5.1 0 00-3.43-2.35"/><path class="a" d="M23.49 8.09a2 2 0 10-2-2 2 2 0 002 2"/><path class="c" d="M25.17 6.05a1.68 1.68 0 11-1.68-1.68 1.68 1.68 0 011.68 1.68m.21 0a1.89 1.89 0 10-1.89 1.89 1.89 1.89 0 001.89-1.89"/></svg>
```

Comment:
Can we replace all `class` by using the values (`fill: #fff`, `fill:#ffcb05`, etc) directly in the svg code?

Reply:
@tauthomas01 done!